arch/src/aarch64/fdt.rs

```diff
@@ -99,6 +99,7 @@ pub fn create_fdt<T: DeviceInfoForFdt + Clone + Debug, S: ::std::hash::BuildHash
     gic_device: &Arc<Mutex<dyn Vgic>>,
     initrd: &Option<InitramfsConfig>,
     pci_space_info: &[PciSpaceInfo],
+    #[cfg(feature = "pci_support")]
     numa_nodes: &NumaNodes,
```

Comment:
are NUMA nodes tied to the PCI feature somehow?

Reply:
No. It is ACPI feature gated. Let me take a look again.